testing/tests/DevExpress.exporter/jspdfParts/jspdf.dataGrid.tests.js

```diff
@@ -21,23 +21,73 @@ QUnit.testStart(() => {
 let helper;
 
 const moduleConfig = {
+    jsPDFUnit: 'pt',
     beforeEach: function() {
         // The transpiling of the script on the drone and locally has differences that affect the imported jsPDF type.
         const _jsPDF = isFunction(jsPDF) ? jsPDF : jsPDF.jsPDF;
-        this.jsPDFDocument = _jsPDF('p', 'pt', 'a4');
+        this.jsPDFDocument = _jsPDF('p', this.jsPDFUnit, 'a4');
 
         helper = new JSPdfDataGridTestHelper(this.jsPDFDocument);
     }
 };
 
-QUnit.module('Scenarios, generate autoTable options', moduleConfig, () => {
+['mm', 'cm', 'em', 'px', 'pt'].forEach((unit) => {
+    const _moduleConfig = $.extend(moduleConfig, {
+        jsPDFUnit: unit
+    });
+
+    QUnit.module('Scenarios, check generated content', _moduleConfig, () => {
+        const getOptions = (context, dataGrid, expectedCustomizeCellArgs, options) => {
```

Comment:
'expectedCustomizeCellArgs' is not used, remove?

Reply:
ok